src/game.cpp

```diff
@@ -3271,6 +3271,35 @@ bool game::handle_action()
                     as_m.text =
                         _("You're engorged to hibernate. The alarm would only attract attention. Enter hibernation?");
                 }
+                // List all active items, bionics or mutations so player can deactivate them
+                std::vector<std::string> active;
+                for ( auto &it : g->u.inv_dump() ) {
+                    if ( it->active ) {
+                        active.push_back( it->tname() );
+                    }
+                }
+                for ( int i = 0; i < g->u.num_bionics(); i++ ) {
+                    bionic &bio = g->u.bionic_at_index(i);
+                    if ( bio.powered && bionics[bio.id]->power_over_time > 0 &&
+                         bio.id != "bio_alarm" ) {
```

Comment:
This isn't ideal--better to have some metric rather than calling by id, saves maintenance--but it's workable for an initial implementation.